src/commands.cpp

```diff
@@ -394,6 +395,18 @@ void AOClient::cmdFlip(int argc, QStringList argv)
     sendServerMessage(sender_name + " flipped a coin and got " + face + ".");
 }
 
+void AOClient::cmdRoll(int argc, QStringList argv)
+{
+    RollType Type = roll;
+    diceThrower(argc, argv, Type);
+}
+
+void AOClient::cmdRollP(int argc, QStringList argv)
+{
+    RollType Type = rollp;
+    diceThrower(argc, argv, Type);
+}
+
```

Comment:
pass the RollType argument like this: `diceThrower(argc, argv, RollType::ROLLP)`